libs/utils/src/http/error.rs

```diff
@@ -28,7 +28,7 @@ pub enum ApiError {
 }
 
 impl ApiError {
-    pub fn into_response(self) -> Response<Body> {
```

Comment:
NIT: Can you elaborate why this change is needed? Previous signature captured the intent of conversion to response by consuming the error. Methods that convert shared references to some other type are usually called `as_foo`, this way it captures the idea that they can be called multiple times on the same object, which as I see it is not needed here

Reply:
I don't understand this change either.

Reply:
I was having use-after-move problems in route_error_handler(), and I fixed / worked around it by using a reference instead. But I figured it out now (Error::downcast() returns the original error in the Err() result, if the downcast fails. I didn't realize that at first). So reverted this back to into_response.